src/components/Molecules/navigation/NavButtons.css

```diff
@@ -2,6 +2,26 @@
 
 }
 
-.nav-buttons {
-    color: #61dafb;
+.prev-button {
```

Comment:
Als zowel de `.prev-button` als de `next-button` dezelfde styling hebben, dan kun je er toch één class van maken en die op beide buttons zetten? Nu doen we dubbel werk :)